guides/release/components-update/conditional-content.md

```diff
@@ -0,0 +1,120 @@
+You can conditionally include some content in the output of a template by using
+Ember's `if` syntax in templates.
+
+Let's take a look at two similar components representing a user's username.
+
+```handlebars {data-filename="app/components/received-message/username.hbs"}
+<h4 class="username">
+  Tomster
+  <span class="local-time">their local time is 4:56pm</span>
+</h4>
+```
+
+```handlebars {data-filename="app/components/sent-message/username.hbs"}
+<h4 class="username">Zoey</h4>
+```
+
+We can use arguments to make these two components dynamic, but the first
+username has extra information in the form of the local time of the user.
+
+Let's say we tried to create a single `address` component.
+
+```handlebars {data-filename="app/components/username.hbs"}
+<h4 class="username">
+  {{@name}}
+  <span class="local-time">their local time is {{@localTime}}</span>
+</h4>
+```
+
+If the `<Username>` tag doesn't specify a `@localTime` argument, we'll end up
+with some extra unneeded text in the output.
+
+What we need is a way to only include the local time text `@locaTime` exists at
+all. We can do this with an `if`:
+
+```handlebars {data-filename="app/components/username.hbs"}
+<h4 class="username">
+  {{@name}}
+  {{#if @localTime}}
+    <span class="local-time">their local time is {{@localTime}}</span>
+  {{/if}}
+</h4>
+```
+
+This the syntax for conditionals in an Ember template. The `#if` means that the
+part after the `{{#if ...}}` is nested inside of the conditional. Just like HTML
+tags continue until closed (`<div>` continues until `</div>`), the content
+nested inside an `#if` continues until `{{/if}}`.
+
+## Inline `if`
+
+Sometimes, you need to place conditional content inside an argument or
+attribute. You can do this with _inline_ `if`. For instance, consider these two
+avatar components
+
+```handlebars {data-filename="app/components/received-message/avatar.hbs"}
+<aside>
+  <div class="avatar is-active" title="Tomster's avatar">T</div>
+</aside>
+```
+
+```handlebars {data-filename="app/components/sent-message/avatar.hbs"}
+<aside class="current-user">
+  <div class="avatar" title="Zoey's avatar">Z</div>
+</aside>
+```
+
+The first component needs to have an `is-active` class applied to the avatar to
+show the active icon. That icon could also change over time, so it probably
+doesn't make sense to try to use `...attributes` to apply it. It's a part of the
+_public API_ of the component, so an argument makes sense.
+
+So, we want to add the `is-active` class if an argument, like say `@isActive`,
+is passed in and is truthy.
```

Comment:
> So, we want to add the `is-active` class if an argument, like say `@isActive`,
is passed in and is truthy.

We want to add the `is-active` class if the argument `@isActive` is passed into the component, and if its value is `true`.

(Have we explained truthiness up until this point yet?)

Reply:
> (Have we explained truthiness up until this point yet?)

No, unfortunately. It's one of my TODOs to figure out where to put it. Would a Zoey Says be good enough around this area?

Reply:
Zoey says would work well, I think 👍